app/models/collection.rb

```diff
@@ -11,6 +11,6 @@ class Collection < ActiveFedora::Base
   prepend OrderAlready.for(:creator)
 
   def remove_featured
```

Comment:
How does this method get called?

Reply:
This is called via some callbacks defined inside the class but outside the scope of this diff.